### PR TITLE
open-cluster-management/backlog#17411 ki filename 2 long

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -148,6 +148,33 @@ Check the role permissions and update appropriately. See link:../access_control/
 [#cluster-management-issues]
 == Cluster management known issues
 
+[#iso-image-name-too-long]
+=== Cluster provisioning with Infrastructure Operator fails
+//2.4:17411
+
+When creating {ocp-short} clusters using the Infrastructure Operator, the file name of the ISO image might be too long. The long image name causes the image provisioning and the cluster provisioning to fail. To determine if this is the problem, complete the following steps: 
+
+. View the bare metal host information for the cluster that you are provisioning by running the following command:
```
Comment (CrystalChun) [inline release_notes/known_issues.adoc:157]: Could we add view & describe? Would that be confusing? The error status below can be found by describing the resource as opposed to just viewing it.
 
View command (already here):
```
oc get bmh -n <cluster provisioning namespace>
```
Describe command:
```
oc describe bmh -n <cluster provisioning namespace> <bmh name>
```

Comment (cadawson) [reply release_notes/known_issues.adoc:157]: Sure. See if this works.

Comment (CrystalChun) [reply release_notes/known_issues.adoc:157]: Looks great, thanks Chris!

**Review by CrystalChun — APPROVED** [review]
Lgtm!

**Pull request merged**